### PR TITLE
Add qiurui as maintainer for orangepizero2

### PR DESCRIPTION
**Pull request by qiurui144**
Added myself as official maintainer for orangepizero2; also added myself in github -> forum name translation table.

### CODE REVIEW

**Comment by EvilOlaf** [comment]
Have you applied as maintainer in forums?
I can barely find any content of yours in forums?
Have you read the board support rules? https://docs.armbian.com/User-Guide_Board-Support-Rules/

**Comment by qiurui144** [comment]
i have communited with [krachlatte] igor
message is https://forum.armbian.com/messenger/165070/

**Comment by EvilOlaf** [comment]
ok.
Btw. even though I am administrator in forums I cannot read messages from individuals.

File: docs/Release_Board-Maintainers.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ The authoritative list of `board.conf` can be found [here](https://github.com/ar
 | orangepiwin          |                |          |
 | orangepizero         | igorpecovnik   |          |
 | orangepizero2        | krachlatte     |          |
+|"                     | qiurui         |          |
```
Comment (igorpecovnik) [inline docs/Release_Board-Maintainers.md:85]: Why its " here?

**Review by igorpecovnik — APPROVED** [review]
@EvilOlaf We spoke yesterday, yes. Happening so fast.

File: docs/Release_Board-Maintainers.md (hunk 1 above)
Comment (EvilOlaf) [reply docs/Release_Board-Maintainers.md]: Meaning maintaining the same board as above

**Comment by EvilOlaf** [comment]
Forum group adjusted

**Pull request merged**